packages/stream-collector-browser/src/index.ts

```diff
@@ -1,9 +1,17 @@
 import { StreamCollector } from "@aws-sdk/types";
 
-export const streamCollector: StreamCollector = (
-  stream: ReadableStream
+export const streamCollector: StreamCollector = async (
+  stream: ReadableStream<Uint8Array>
 ): Promise<Uint8Array> => {
-  return new Response(stream)
-    .arrayBuffer()
-    .then(arrayBuffer => new Uint8Array(arrayBuffer));
+  const res: number[] = [];
+  const reader = stream.getReader();
+  let isDone = false;
+  while(!isDone) {
+    const { done, value } = await reader.read();
+    if(value) {
+      res.push(...value);
```

Comment:
all, let me know if there's a better way to combine multiple UintArray's into one.

Reply:
nvm, found it. testing & pushing patch shortly.